### PR TITLE
Some updates to v3

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/studying.md [hunk 1]
```diff
@@ -11,9 +11,10 @@ Study in Anki is limited to the currently selected deck as well as any
 subdecks it contains.
 
 On the decks screen, your decks will be displayed in a list. There are
-two number columns, 'due' and 'new'. 'Due' is the count of waiting
-reviews and cards currently in learning. 'New' is the number of new
-cards that are ready to be learnt that day.
+three columns. 'Due' is the count of waiting
+reviews. 'New' is the number of new
+cards that are ready to be learnt that day. The third column shows
+the number of cards currently in learning.
```
Comment (dae) [inline src/studying.md:17]: Is  this order correct?

File: src/studying.md [hunk 2]
```diff
@@ -25,11 +26,11 @@ can press the ‘s’ key to study the currently selected deck.)
...
 delete a deck, change its options, or [export](exporting.md) it.
 
-With the old scheduler, when a deck has subdecks, the cards will appear from [each deck in
+With the v1 scheduler, when a deck has subdecks, the cards will appear from [each deck in
 turn](studying.md#display-order).
 
-With the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
-when a deck has subdecks, reviews are now taken from all children decks
+From the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+when a deck has subdecks, reviews are taken from all children decks
```
Comment (dae) [inline src/studying.md:33]: In v3, doesn't this depend on the configured options?

File: src/studying.md [hunk 3]
```diff
@@ -93,9 +94,9 @@ card will be shown again the next day, then at increasingly long delays
 
 **Easy** immediately converts the card into a review card, even if there
 were steps remaining. By default, the card will be shown again 4 days
-later, and then at increasingly long delays. In the old scheduler, the "Easy" button will not be
+later, and then at increasingly long delays. In the v1 scheduler, the "Easy" button will not be
 shown if you are in relearning mode and it would give the same interval
-as “Good.” With the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+as “Good.” With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
```
Comment (dae) [inline src/studying.md:99]: v2+?

File: src/studying.md [hunk 4]
```diff
@@ -140,30 +141,33 @@ should find yourself answering 'Good' instead.
...
 you can turn them off in Anki’s preferences.
 
-In the old scheduler, the numbers count _reviews_ needed to finish all the
+In the v1 scheduler, the numbers count _reviews_ needed to finish all the
 cards in that queue, not the number of _cards_. If you have multiple
 steps configured for lapsed cards, the number will increase by more than
 one when you fail a card, since that card needs to be shown several times.
 
-In the new scheduler, the numbers count _cards_, so the number will always
+From the v2scheduler, the numbers count _cards_, so the number will always
```
Comment (dae) [inline src/studying.md:159]:
```suggestion
From the v2 scheduler, the numbers count _cards_, so the number will always
```

File: src/studying.md [hunk 5]
```diff
@@ -140,30 +141,33 @@ should find yourself answering 'Good' instead.
...
 estimates, you can disable them in Anki’s [preferences](preferences.md).
 
-Anki additionally adds a small amount of random variation to the next
+Anki additionally adds a small amount of random variation (Fuzz Factor) to the next
 due times, in order to prevent cards that were introduced together and
-always rated the same from always staying next to each other. This
-variation is not shown on the time estimates but will be applied after
-selecting the button.
+always rated the same from always staying next to each other. From 2.1.45+, 
+this variation is reflected on the answer buttons, instead of only being
```
Comment (dae) [inline src/studying.md:169]:
```suggestion
this variation is reflected on the answer buttons.
```

**Review by dae — COMMENTED** [review]
Thanks cqg! Some comments below.

File: src/studying.md (hunk 1 above)
Comment (cqg2) [reply src/studying.md]: Corrected. The second column is unlabelled, perhaps label it as "Learn" could be helpful, specially for new users.

File: src/studying.md (hunk 2 above)
Comment (cqg2) [reply src/studying.md:33]: Better this way?

```
- With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
when a deck has subdecks, reviews are taken from all children decks
at once. The review limit of the child decks is ignored - only the limit of the
deck you clicked on applies.

- With the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html)
each child deck's limit is also enforced, and you do not need to see the cards 
in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual. 
```

File: src/studying.md (hunk 3 above)
Comment (cqg2) [reply src/studying.md]: Corrected

File: src/studying.md [hunk 6]
```diff
@@ -93,9 +98,9 @@ card will be shown again the next day, then at increasingly long delays
 
 **Easy** immediately converts the card into a review card, even if there
 were steps remaining. By default, the card will be shown again 4 days
-later, and then at increasingly long delays. In the old scheduler, the "Easy" button will not be
+later, and then at increasingly long delays. In the v1 scheduler, the "Easy" button will not be
 shown if you are in relearning mode and it would give the same interval
```
Comment (dae) [inline src/studying.md:102]:
```suggestion
shown if you are in relearning mode as it would give the same interval
```

File: src/studying.md [hunk 7]
```diff
@@ -140,30 +145,33 @@ should find yourself answering 'Good' instead.
...
-In the new scheduler, the numbers count _cards_, so the number will always
+From the v2 scheduler, the numbers count _cards_, so the number will always
 increase by one regardless of the steps remaining.
 
 When the answer is shown, Anki shows an estimate of the next time a card
 will be shown above each button. If you’d prefer not to see the
 estimates, you can disable them in Anki’s [preferences](preferences.md).
 
-Anki additionally adds a small amount of random variation to the next
+Anki additionally adds a small amount of random variation (Fuzz Factor) to the next
```
Comment (dae) [inline src/studying.md:170]:
```suggestion
Anki additionally adds a small amount of random variation ("fuzz") to the next
```

File: src/studying.md [hunk 8]
```diff
@@ -140,30 +145,33 @@ should find yourself answering 'Good' instead.
...
 due times, in order to prevent cards that were introduced together and
-always rated the same from always staying next to each other. This
-variation is not shown on the time estimates but will be applied after
-selecting the button.
+always rated the same from always staying next to each other. From 2.1.45+,
```
Comment (dae) [inline src/studying.md:172]:
```suggestion
always rated the same from always staying next to each other. In the v3 scheduler, 
```

File: src/studying.md [hunk 9]
```diff
@@ -140,30 +145,33 @@ should find yourself answering 'Good' instead.
...
 
-Anki additionally adds a small amount of random variation to the next
+Anki additionally adds a small amount of random variation (Fuzz Factor) to the next
 due times, in order to prevent cards that were introduced together and
-always rated the same from always staying next to each other. This
-variation is not shown on the time estimates but will be applied after
-selecting the button.
+always rated the same from always staying next to each other. From 2.1.45+, 
+this variation is reflected on the answer buttons.
+applied when answering, as in previous versions.
```
Comment (dae) [inline src/studying.md:174]: Delete this line?

File: src/studying.md [hunk 10]
```diff
@@ -25,14 +26,18 @@ can press the ‘s’ key to study the currently selected deck.)
...
-With the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
-when a deck has subdecks, reviews are now taken from all children decks
+- With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+when a deck has subdecks, reviews are taken from all children decks
 at once. The review limit of the child decks is ignored - only the limit of the
 deck you clicked on applies.
 
+- With the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html)
+each child deck's limit is also enforced, and you do not need to see the cards 
+in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual.
```
Comment (dae) [inline src/studying.md:39]:
```suggestion
in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual for more information.
```

**Review by dae — COMMENTED** [review]
Thanks again cqg, a few more comments.

File: src/studying.md (hunk 9 above)
Comment (cqg2) [reply src/studying.md]: Deleted, sorry!

**Comment by cqg2** [comment]
Updated with corrections. Thanks!

**Comment by dae** [comment]
Thanks!

**Pull request merged**